frontend/appflowy_flutter/lib/workspace/presentation/home/menu/menu.dart

```diff
@@ -105,6 +111,50 @@ class HomeMenu extends StatelessWidget {
     );
   }
 
+  Widget _renderFavorites(BuildContext context) {
+    return BlocBuilder<FavoriteBloc, FavoriteState>(
+      builder: (context, state) {
+        return state.views.isNotEmpty
```

Comment:
I would prefer not to use the ternary operator on such a large widget. Up to preference though.
```
if (state.views.isEmpty) {
  return const SizedBox.shrink();
}

return ExpandableTheme(
```

Reply:
Actually this code is deprecated, all the logic for favorites and views is now under sidebar/sidebar.dart and sidebar/folder/favorites_folder.dart.

I think removal of deprecated code will be handled by lucas after making sure it does not impact anything else